src/main/java/com/dsec/backend/controller/RepoController.java

```diff
@@ -37,4 +40,12 @@ public ResponseEntity<EmptyDTO> triggerHook(@PathVariable("id") long id,
         repoService.triggerHook(id, jwt);
         return ResponseEntity.ok(new EmptyDTO());
     }
+
+    @GetMapping("/{repo}/jobs")
+    public ResponseEntity<List<Job>> getJobs(@PathVariable("repo") long id, @AuthenticationPrincipal Jwt jwt)
```

Comment:
You are taking repo as an id. It would be helpful if you name such variables with an Id in the name. Something like repoId so it is clear what should go there.